src/js/controllers/modals/walletBalance.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+angular.module('copayApp.controllers').controller('walletBalanceController', function($scope) {
+
+  $scope.close = function() {
+    $scope.walletBalanceModal.hide();
```

Comment:
Please, if the controller will not have more logic just use the next line on *Close* button in the HTML tag

> ng-click="walletBalanceModal.hide()"

And delete the controller